### PR TITLE
Update OpenTelemetry to 1.4.0

### PR DESCRIPTION
**Pull request by carlosalberto**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,8 +24,8 @@
     <maven.compiler.target>1.8</maven.compiler.target>
     <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <opentelemetry.version>1.2.0</opentelemetry.version>
-    <opentelemetry.instrumentation.version>1.2.0</opentelemetry.instrumentation.version>
+    <opentelemetry.version>1.4.1</opentelemetry.version>
```
Comment (codeboten) [inline pom.xml:27]: 1.4.1?

**Review by codeboten — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (carlosalberto) [reply pom.xml:27]: That's correct as Java Instrumentation uses OTel Java 1.4.1, see: https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.4.0

Comment (codeboten) [reply pom.xml:27]: Ah, in that case should all references to OpenTelemetry be updated to 1.4.1 instead?

Comment (carlosalberto) [reply pom.xml:27]: To OTel, yes - to OTel Instrumentation, 1.4.0 ;)

**Pull request merged**